### ISSUE TITLE
something wrong with val_score_best

### ISSUE THREAD

**Issue by jvschleicher** [lcmd-epfl/EquiReact#54]
[Epoch 12; Iter    65/   85] train: loss: 0.0962532                                                                                          │
[Epoch 12; Iter    85/   85] train: loss: 0.0166987                                                                                          │
[Epoch 12] mae: 1.116529 val loss: 0.145796                                                                                                  │
Epochs with no improvement: [ 0 ] and the best mae was in 12                                                                                 │
Statistics on val_best_checkpoint                                                                                                            │
mae: 1.1165286655164222                                                                                                                      │
MSELoss: 0.14579571868208321                                                                                                                 │
mean_pred: 0.07593964040279388                                                                                                               │
std_pred: 1.0041318373246626                                                                                                                 │
mean_targets: 0.12800395691936667                                                                                                            │
std_targets: 1.0936204140836543                                                                                                              │
Evaluating on test, with test size:  84                                                                                                      │
Statistics on test_split_1                                                                                                                   │
mae: 1.0544128565778936                                                                                                                      │
MSELoss: 0.10409140400588512                                                                                                                 │
mean_pred: -0.1278844028711319                                                                                                               │
std_pred: 0.8951043351130052                                                                                                                 │
mean_targets: -0.09881682152097876                                                                                                           │
std_targets: 0.9225046309557828                                                                                                              │
wandb: Waiting for W&B process to finish... (success).                                                                                       │
wandb: - 0.009 MB of 0.009 MB uploaded (0.000 MB deduped)^Mwandb: \ 0.009 MB of 0.009 MB uploaded (0.000 MB deduped)^Mwandb: | 0.009 MB of 0.│
009 MB uploaded (0.000 MB deduped)^Mwandb: / 0.009 MB of 0.009 MB uploaded (0.000 MB deduped)^Mwandb: - 0.009 MB of 0.009 MB uploaded (0.000 │
MB deduped)^Mwandb:                                                                                                                          │
wandb: Run summary:                                                                                                                          │
wandb:          epoch 12                                                                                                                     │
wandb:      test_rmse 1.35409                                                                                                                │
wandb:     test_score 1.05441                                                                                                                │
wandb:     train loss 0.0167                                                                                                                 │
wandb:       val_loss 0.1458                                                                                                                 │
wandb:      val_score 1.11653                                                                                                                │
wandb: val_score_best 1.1428
